Essentials/src/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -806,6 +806,13 @@ public void onPlayerFishEvent(final PlayerFishEvent event) {
         user.updateActivityOnInteract(true);
     }
 
+    @EventHandler(priority = EventPriority.LOW)
+    public void onPlayerPickupFireball(final PlayerPickupArrowEvent event) {
+        if (event.getArrow().hasMetadata("Ess_Fireball_Projectile")) {
```

Comment:
Is there a common place where this string can live with references back? Sentinel string constants in multiple places scare me :(

Reply:
Feel free to do it yourself, this was just a dirty fix to show it wasn’t as hard as pineapple guy suggested.

Reply:
Feel free to close the issue if you aren't planning on working on this further.